ray-curator/ray_curator/config.py

```diff
@@ -1,3 +1,17 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Wait should this file exist?

Reply:
Not sure if it's being used anywhere tbh...